### ISSUE TITLE
Best way to clarify a subgenus name?

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#1060]
This has been requested by the Sarcophagidae recording scheme. The name "Sarcophaga" is used as a genus name, and as a subgenus. At the moment, there is no way to disinguish these two taxon ranks during data entry in iRecord:

![image](https://user-images.githubusercontent.com/16039111/118497660-2f464100-b71d-11eb-9b79-867c4e658130.png)

Which of these options is the best:
1. Make the subgenus unavailable for data entry (not ideal, the scheme would prefer to have both ranks available)
2. Add the taxon rank as a vernacular name, so that they display as Sarcophaga (genus) and Sarcophaga (subgenus)
3. Edit the text string for the subgenus so that it has brackets around it (but not sure how this will affect the way it is displayed elsewhere)

Option 2 seems simplest to me.

**Comment by johnvanbreda** [comment]
Yes, option 2 seems best to me to. It needs the taxon_rank field added to the taxon names search cache table but that is not too hard to do. 
We need to think about when to display the rank. Presumably not for a species, but what about subspecies, or other higher taxa? Or do we only display the rank when there is a clash of names in the list?

**Comment by kitenetter** [comment]
I have found further examples of subgenus names being identical to genus names. Manually changing the vernacular names in iRecord (option 2 above) is a solution but not a very sustainable one.

Should we look at displaying the rank within iRecord, or should we consult with Chris to see if there is a better way of displaying the names within UKSI?

As for when a rank needs to be displayed, as far as I know the only time this has caused real confusion is with subgenus names, because these can be indistinguishable from genus names. Subspecies are distinguishable because they are presented as trinomials - still confusing for novices but distinguishable when you know what they are.

**Comment by johnvanbreda** [comment]
I think we can display the rank only if there is a duplication in the list shown which the rank helps to disambiguate. So, you wouldn't need a fake vernacular, we'd just use the rank itself (but I'd need to tweak some index tables for this to work).
